earth_enterprise/src/fusion/rasterfuse/TmeshPackgenTraverser.cpp

```diff
@@ -91,7 +95,10 @@ void TmeshWorkItem::DoWork(TmeshPrepItem *prep) {
                           subcol * sampleTilespace.tileSize),
          piece->targetAddr,
          piece->compressed,
-         kCRC32Size /* reserve size */);
+         kCRC32Size /* reserve size */,
+         decimate_,
+         decimation_threshold_
```

Comment:
would be good to have comments to explain the decimate_ 
 and decimate_threshold_